src/kaleidoscope/key_defs.h

```diff
@@ -255,14 +260,19 @@ typedef kaleidoscope::Key Key_;
 #define KEY_LEFT_FN2 0xff
 #define Key_LFN2 Key(KEY_LEFT_FN2, KEY_FLAGS)
 
+#define SYSTEM_KEY(code, hid_type) \
+  Key(code, SYNTHETIC | IS_SYSCTL | (hid_type & HID_TYPE_MASK))
```

Comment:
I've added back in the HID Usage Type parameter. I'm using the `HID_TYPE_MASK` to ensure that only the allowed bits can be set in the `flags` byte.